vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go

```diff
@@ -226,7 +226,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdUncordon(f, out))
 
 	cmds.AddCommand(NewCmdAttach(f, in, out, err))
-	cmds.AddCommand(NewCmdExec(f, in, out, err))
+  cmds.AddCommand(NewCmdExec("kubectl", f, in, out, err))
```

Comment:
Thanks for catching this, the gofmt plugin I have did not seem to catch it either. Fixed!